### PR TITLE
Expose Requirement.marker to python

### PR DESCRIPTION
**Pull request by charbeljc**
Hi, playing around with your things :-)
Regards,
Charbel

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -210,6 +210,17 @@ impl Requirement {
...
 
+    /// The marker expression such as  `python_version > "3.8"` in
+    /// `requests [security,tests] >= 2.8.1, == 2.8.* ; python_version > "3.8"`
+    #[getter]
+    pub fn marker(&self) -> Option<String> {
+        if self.marker.is_some() {
+            Some(self.marker.as_ref().unwrap().to_string())
+        } else {
+            None
+        }
```
Comment (konstin) [inline src/lib.rs]: You can use [map](https://doc.rust-lang.org/nightly/std/option/enum.Option.html#method.map) here to make this a single line call

**Review by konstin — COMMENTED** [review]
Could you please add a test in [test_pep508.py](https://github.com/konstin/pep508_rs/blob/main/test/test_pep508.py)?

File: src/lib.rs (hunk 1 above)
Comment (charbeljc) [reply src/lib.rs]: oh, rust is nice :-)

Comment (charbeljc) [reply src/lib.rs]: added a minimal test

**Comment by konstin** [comment]
Please run `cargo fmt` in the project root, the [lint CI job](https://github.com/konstin/pep508_rs/actions/runs/5089538229/jobs/9148341751) is currently failing

**Comment by charbeljc** [comment]
Ran cargo fmt. This should be ok now

**Comment by konstin** [comment]
thanks!

**Pull request merged**